jsonp/package.json

```diff
@@ -10,24 +10,22 @@
     "url": "https://github.com/cyclejs/jsonp"
   },
   "dependencies": {
-    "@cycle/xstream-adapter": "3.x.x",
+    "@cycle/run": "^1.0.0-rc.7",
     "jsonp": "0.2.0"
   },
   "peerDependencies": {
     "xstream": "*"
   },
   "devDependencies": {
-    "eslint": "1.0.0",
-    "eslint-config-cycle": "3.2.0",
-    "eslint-plugin-cycle": "1.0.2",
-    "eslint-plugin-no-class": "0.1.0",
-    "xstream": "8.x.x"
+    "xstream": "10.x.x"
   },
   "browserify-shim": {
     "xstream": "global:xstream"
   },
   "scripts": {
-    "lint": "eslint src/",
+    "compile": "../node_modules/.bin/tsc",
+    "prepublish": "npm run compile",
```

Comment:
I'll do this myself, but notice how we call the script `lib`, not `compile`, in other packages.